src/main/java/prefs/Preferences.java

```diff
@@ -15,23 +19,26 @@
  * It cannot be as we need to mock it.
  */
 public class Preferences { // NOPMD
+
+    private static final Logger logger = LogManager.getLogger(Preferences.class.getName());
+
     public static final String DIRECTORY = "settings";
 
     // Standard config filenames used for application and testing
-    public static final String GLOBAL_CONFIG_FILE = "global.json";
-    public static final String TEST_CONFIG_FILE = "test.json";
+    public static final String SESSION_CONFIG_FILE = "global.json";
+    public static final String TEST_SESSION_CONFIG_FILE = "test.json";
 
-    private final ConfigFileHandler fileHandler;
+    private final ConfigFile sessionConfigFile;
 
-    public GlobalConfig global;
+    public GlobalConfig sessionConfig;
```

Comment:
I don't think `sessionConfig` should be public, but I saw a test where it mocks this config, which needs a direct setting of this variable. This is a bit troublesome, but can you change this to `private` and use reflection for the test case?